<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Region2dSettingEditWidget</class>
 <widget class="QWidget" name="Region2dSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="fullRadioButton">
     <property name="text">
      <string>Full Region</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="activeRadioButton">
     <property name="text">
      <string>Active Region</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="customRadioButton">
     <property name="text">
      <string>Custom Region</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="RegionRange2dSettingEditWidget" name="rangeWidget" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegionRange2dSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/region/regionrange2dsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>customRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>rangeWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
